db/schema.rb

```diff
@@ -729,6 +729,7 @@
     t.string "webhook_url", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "signature", default: 0
```

Comment:
Probably want to rename this to something a bit more clear? Maybe `signature_algo`?